firstvoices/backend/models/page.py

```diff
@@ -38,7 +38,9 @@ class Meta:
         db_index=True,
         unique=False,
     )
-    subtitle = models.CharField(blank=True, max_length=constants.DEFAULT_TITLE_LENGTH)
+    subtitle = models.CharField(
+        null=True, blank=True, max_length=constants.DEFAULT_TITLE_LENGTH
+    )
```

Comment:
Please don't allow nulls in CharFields: https://docs.djangoproject.com/en/4.2/ref/models/fields/#null

If we want to allow the frontend to send null values, we need to handle that in the serializer instead (convert null to "" when saving). I'll log a ticket to do that for all string fields in all the serializers eventually, but feel free to set it up for this one API as part of this ticket, or just ask the frontend to send "" for now